community/cypher/cypher-compiler-2.2/src/main/scala/org/neo4j/cypher/internal/compiler/v2_2/ast/convert/commands/ExpressionConverters.scala

```diff
@@ -47,6 +47,8 @@ object ExpressionConverters {
       case e: ast.NotEquals => e.asCommandNotEquals
       case e: ast.RegexMatch => e.asCommandRegex
       case e: ast.In => e.asCommandIn
+      case e: ast.Like => e.asCommandRegex
+      case e: ast.NotLike => e.asCommandNegatedRegex
```

Comment:
@davidegrohmann The `NotEquals` -> `Not(Equals)` transformation is done on the commands right? There is no `Like` on the command level (yet?). We transform to `RegularExpression` and `Not(RegularExpression)` so I am not sure exactly what kind of rewrite you propose?